### PR TITLE
Use correct admonitions

### LINKED ISSUE #26: Use correct admonitions
The current style of Notes etc does not look nice. There must be a wrong use of admonitions. Please check how to do this correctly.

## Read Existing Documentation

I have checked the following places for the error:

- [ ] [OpenMS Documentation Troubleshooting]()
- [ ] [OpenMS API Developer FAQs]()

## Create OpenMS Bug Report

I have created the bug report in OpenMS for any pipeline errors.

- [ ] [OpenMS Open Issues](https://github.com/OpenMS/OpenMS/issues)

## Description of the documentation bug

- [ ] I have attached screenshots of the incorrect documentation

## OpenMS Installation

- Version:

### PR DESCRIPTION
**Pull request by ghost**
## Describe the change
Made one spelling correct. The other changes relate to adding the agreed upon admonition syntax. 
Closes #26.

## PR checklist

- [x] I have added description of the change I'm proposing in the OpenMS Documentation.
- [x] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [ ] I have attached a screenshot of the relevant area after this change.
- [ ] `CHANGELOG.md` is updated.
- [ ] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

File: docs/installations/installation-on-macos.md [hunk 1]
```diff
@@ -41,9 +41,11 @@ Make sure `<OpenMS-PATH>` points to the folder where OpenMS is installed locally
 
    Since macOS Catalina (maybe also Mojave) notarized apps and executables are mandatory.
 
-   > **_NOTE:_** Although there is a lot of effort in signing and notarizing everything, it seems like openms software
+   ```{note}
```
Comment (tapaswenipathak) [inline docs/installations/installation-on-macos.md]: let's make this:

important.

File: docs/installations/installation-on-windows.md [hunk 2]
```diff
@@ -7,8 +7,10 @@ To Install the binary package of OpenMS & TOPP:
 2. Run the installer under the user account that later runs OpenMS. It is not advised to use admin account for
    installation. When asked for an admin authentication, please enter the credentials.
 
-> **_NOTE:_** The windows binary version works with most versions of windows from Win7 to Win10 (older versions might
+```{note}
```
Comment (tapaswenipathak) [inline docs/installations/installation-on-windows.md]: Tip.

File: docs/tutorials/TOPP/feature-detection.md [hunk 3]
```diff
@@ -14,14 +14,16 @@ wavelet**, which has been tailored to the detection of isotopic patterns followi
 information about the theory behind this technique, please refer to Hussong et al.: "Efficient Analysis of Mass
 Spectrometry Data Using the Isotope Wavelet" (2007).
 
-Please note that this algorithm features no "modelling stage", since the structure of the isotopic pattern is explicitly
+```{note}
```
Comment (tapaswenipathak) [inline docs/tutorials/TOPP/feature-detection.md]: attention.

File: docs/tutorials/TOPP/feature-grouping.md [hunk 4]
```diff
@@ -29,5 +29,7 @@ To differentially quantify the features of two or more label-free HPLC-MS map.
 
 ![TOPP labelfree quant](../../images/tutorials/topp/TOPP_labelfree_quant.png)
 
-> **_NOTE:_** This algorithm assumes that the retention time axes of all input maps are very similar. To correct for
+```{note}
```
Comment (tapaswenipathak) [inline docs/tutorials/TOPP/feature-grouping.md]: tip

File: docs/tutorials/TOPP/general-introduction.md [hunk 5]
```diff
@@ -61,10 +61,11 @@ can contain the configuration of one or several TOPP tools.
 The following examples will give an overview of how TOPP tools can be chained in order to create analysis pipelines. INI
 files are the recommended way to store all settings of such a pipeline in a single place.
 
-Note that the issue of finding suitable parameters for the tools is not addressed here. For problems during the
+```{note}
```
Comment (tapaswenipathak) [inline docs/tutorials/TOPP/general-introduction.md]: attention

File: docs/installations/installation-on-macos.md (hunk 1 above)
Comment (ghost) [reply docs/installations/installation-on-macos.md]: done

File: docs/installations/installation-on-windows.md (hunk 2 above)
Comment (ghost) [reply docs/installations/installation-on-windows.md]: done

File: docs/tutorials/TOPPAS/examples.md [hunk 6]
```diff
@@ -52,8 +52,10 @@ text based format using the TextExporter for further processing (e.g., in Micros
 The results can be opened in TOPPView. The next figures show the results in 2D and 3D view, together with the feature
 intermediate results.
 
-> **_NOTE:_** The intensities and retention times are slightly different between the runs. To correct for retention
+```{note}
```
Comment (tapaswenipathak) [inline docs/tutorials/TOPPAS/examples.md]: tip

File: docs/tutorials/TOPP/feature-detection.md (hunk 3 above)
Comment (ghost) [reply docs/tutorials/TOPP/feature-detection.md]: done

File: docs/tutorials/TOPP/feature-grouping.md (hunk 4 above)
Comment (ghost) [reply docs/tutorials/TOPP/feature-grouping.md]: done

File: docs/tutorials/TOPP/general-introduction.md (hunk 5 above)
Comment (ghost) [reply docs/tutorials/TOPP/general-introduction.md]: done

File: docs/tutorials/TOPPAS/examples.md (hunk 6 above)
Comment (ghost) [reply docs/tutorials/TOPPAS/examples.md]: done

**Review by tapaswenipathak — APPROVED** [review]
(this. should. work. nice.)

**Pull request merged**